File: C++/1080.cpp
#include <iostream>


int main()
{
    int values[100];


    for (int i = 0; i < 100; i++ )
    {
        std::cin >> values[i];
    }

    int precessor = 0;
    int pos = 0;
    for (int i = 0; i < 100; i++)
    {
        if (values[i] > precessor)
        {
            precessor = values[i];
            pos = i + 1;
        }
    }

    std::cout << precessor << std::endl;
    std::cout << pos << std::endl;

    


}

File: Python/1080.py
values = []

for i in range(100):
    values.append(int(input()))

precessor = 0
pos = 0

for i in range(100):
    if values[i] > precessor:
        precessor = values[i]
        pos = i + 1

print(precessor)
print(pos)
